.github/workflows/Pyodide.yml

```
@@ -0,0 +1,72 @@
+name: DuckDB-Pyodide
+on:
+  workflow_call:
+    inputs:
+      override_git_describe:
+        type: string
+      git_ref:
+        type: string
+      skip_tests:
+        type: string
+  workflow_dispatch:
+    inputs:
+      override_git_describe:
+        type: string
+      git_ref:
+        type: string
+      skip_tests:
+        type: string
+  repository_dispatch:
+  push:
+    branches:
+      - "**"
+      - "!main"
+      - "!feature"
+    paths-ignore:
+      - "**.md"
+      - "examples/**"
+      - "test/**"
+      - "tools/**"
+      - "!tools/pythonpkg/**"
+      - ".github/patches/duckdb-wasm/**"
+      - ".github/workflows/**"
+      - "!.github/workflows/Pyodide.yml"
+
+  pull_request:
+    types: [opened, reopened, ready_for_review]
+    paths-ignore:
+      - "**.md"
+      - "examples/**"
+      - "test/**"
+      - "tools/**"
+      - "!tools/pythonpkg/**"
+      - ".github/patches/duckdb-wasm/**"
+      - ".github/workflows/**"
+      - "!.github/workflows/Pyodide.yml"
```

```suggestion
  push:
    branches:
      - "**"
      - "!main"
      - "!feature"
    paths-ignore:
      - "**"
      - "!.github/workflows/Pyodide.yml"

  pull_request:
    types: [opened, reopened, ready_for_review]
    paths-ignore:
      - "**"
      - "!.github/workflows/Pyodide.yml"
```

This is probably enough, where this is run:
* on a daily basis (as part of the repository_dispatch-ed nightly runs)
* when there are modifications to this workflow
* when invoked explicitly